### PR TITLE
First pass of common SCPs 

### PR DESCRIPTION
**Pull request by dynamike**
This is the initial pass with making this module functional for using in AWS Organizations. It's inspired by asecure.cloud which has some good default SCPs we would likely want to apply to any AWS Org we manage. The initial set of SCPs provide the following 
* Deny the root user from taking any action
* Deny the ability for an AWS account to leave an AWS organization
* Deny the ability to create IAM users and access keys in an AWS account
* Deny the ability to delete KMS keys
* Deny the ability to delete Route53 zones
* Deny all access to an AWS account

Each SCP can be attached to a list of Organizational Units(OU).

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -1 +1,162 @@
...
+  statement {
+    actions   = ["*"]
+    resources = ["*"]
+    condition {
+      test     = "StringLike"
+      variable = "aws:PrincipalArn"
+      values   = ["arn:aws:iam::*:root"]
+    }
+  }
+}
```
Comment (cblkwell) [inline main.tf:15]: Is the thought you'd use this on the child accounts?

**Review by cblkwell — APPROVED** [review]
These seem fine, but I don't know how useful some of them actually will be in practice.

**Review by esacteksab — APPROVED** [review]
I'm left wondering how this might impact the `trussworks-ci` account or the sandbox account. I guess only one way to find out.

File: main.tf (hunk 1 above)
Comment (dynamike) [reply main.tf:15]: yeah, there should be no reason to use the root user in any of the child accounts.

**Comment by dynamike** [comment]
@esacteksab The example is the README is very similar setup to what I have layed out for the Truss org. trussworks-ci sits in the `id_destination` OU as does trussworks-sandbox.

**Comment by dynamike** [comment]
a tacit endorsement from @cblkwell

**Pull request merged**